src/options/components/form.test.tsx

```diff
@@ -87,7 +89,9 @@ describe("form", () => {
 
     expect(checkbox).not.toBeChecked();
     expect(format).toHaveBeenLastCalledWith(expect.any(Object), false);
-    expect(screen.getByText("formatted-commit (false)")).toBeInTheDocument();
+    await waitFor(() => {
+      expect(screen.getByText("formatted-commit (false)")).toBeInTheDocument();
```

Comment:
🥇  gold! I previously changed it to `findByText`, so that it waits - but no luck 😢